deploy/ocm-agent-operator-managednotifications/10-managednotifications-pdb.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: ocmagent.managed.openshift.io/v1alpha1
+kind: ManagedNotification
+metadata:
+  name: sre-pdb-managed-notifications
+  namespace: openshift-ocm-agent-operator
+spec:
+  notifications:
+    - activeBody: |-
+        The pod disruption budget is below the minimum disruptions allowed level and is not satisfied. The number of current healthy pods is less than the desired healthy pods. For more information, please refer to https://github.com/openshift/runbooks/blob/master/alerts/cluster-kube-controller-manager-operator/PodDisruptionBudgetLimit.md
+      name: PodDisruptionBudgetLimit
+      resendWait: 24
+      severity: Warning
```

Comment:
I haven't dug into how `severity` casing is processed, but checking [OCP CI for 4.14.0-rc.1][1]:

```console
$ curl -s https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/logs/periodic-ci-openshift-release-master-nightly-4.14-e2e-aws-ovn-serial/1702743868887273472/artifacts/e2e-aws-ovn-serial/gather-must-gather/artifacts/must-gather.tar | tar xOz quay-io-openshift-release-dev-ocp-v4-0-art-dev-sha256-1d585f4a85ad98d37f969632b62406b061032423a31e2b7df971ef51cfbfe8e5/monitoring/prometheus/rules.json | yaml2json | jq -r '.data.groups[].rules[].labels.severity' | sort | uniq -c | sort -n
      1 none
     17 info
     35 critical
    163 warning
    231 null
```

and looking at the [alerting consistency guide][2], I'd have expected `warning` instead of `Warning` here.

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-release-master-nightly-4.14-e2e-aws-ovn-serial/1702743868887273472
[2]: https://github.com/openshift/enhancements/blob/158111ce156aac7fa6063a47c00e129c13033aec/enhancements/monitoring/alerting-consistency.md#style-guide